resources/gather-deploy-images.sh

```diff
@@ -12,6 +12,8 @@ function get-images-per-env() {
 	
 	IMAGE_PATH='.spec.template.spec.containers[0].image'
 	component_name=$(yq .metadata.name application.yaml)
+
+	IMAGES_FILE=$HOMEDIR/all-images.txt
```

Comment:
Does it make sense to contribute to [upstream](https://github.com/redhat-appstudio/tssc-sample-pipelines/blob/main/pac/tasks/gather-deploy-images.yaml) tekton tasks and then regenerate this file to have less manual divergence?

Reply:
Yeah, I could do that.

Reply:
I'd probably not want to change the location, but the refactor to introduce the IMAGES_FILE var seems like a reasonable idea.

Reply:
See https://github.com/konflux-ci/build-definitions/pull/1240 , is that what you were imagining?

Reply:
Merged now.